File: app/src/main/java/com/velaphi/tabby/data/remote/ApiService.java
package com.velaphi.tabby.data.remote;

import com.velaphi.tabby.data.model.Image;

import java.util.List;

import retrofit2.Call;
import retrofit2.http.GET;

public interface ApiService {
    @GET("images/get?format=json&results_per_page=100&size=small&type=png")
    Call<List<Image>> getListOfImages();
}

File: app/src/main/java/com/velaphi/tabby/data/api/ApiService.kt
package com.velaphi.tabby.data.api

import com.velaphi.tabby.data.model.Response
import retrofit2.http.GET

interface ApiService {
    @GET("images/get?format=xml&results_per_page=100&size=medium&type=jpg")
    suspend fun getImages(): Response
}